features/object-position.yml

```
@@ -0,0 +1,7 @@
+name: object-position
+description: The `object-position` CSS property places images, videos, and other replaced elements within their container.
```

I worry a little that "container" implies "containing element" and not the element itself. We've used the term "box" elsewhere I think it might do OK here.

```suggestion
description: The `object-position` CSS property places images, videos, and other replaced elements within their boxes.
```

Or we could spell it out in more words:

```suggestion
description: The `object-position` CSS property places an image, video, or other replaced element within the element's content boundaries.
```